### PR TITLE
update integration tests

### PR DESCRIPTION
**Pull request by CatalinFaurPaul**
create new integration tests

## Type
- [x] Integration tests
  
## Proposed Changes
-  fixed apr value #514 
-  Nfts with owner pagination #515 
-  Account tokens from Elasticsearch if Smart Contract #519 
- and more

### CODE REVIEW

File: src/test/integration/transaction.get.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        expect(result.hasOwnProperty("returnMessage")).toBeTruthy();
+      }
+    });
+
+    it("should return transaction from gateway with queryInElastic = true", async () => {
+      const txHash: string = "51ffbf3d27e06fd509c510ef0ff1ea7329359dba89c05aeec333de52a405664d";
+      const results = await transactionGetService.tryGetTransactionFromGateway(txHash, true);
+
+      if (!results) {
+        throw new Error("Properties are not defined");
```
Comment (CatalinFaurPaul) [inline src/test/integration/transaction.get.e2e-spec.ts:64]: misspelled word

File: src/test/integration/transaction.get.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      expect(results.logs).toBeDefined();
+      expect(results.logs?.address).toStrictEqual("erd1qqqqqqqqqqqqqpgqy47ztjxnnqzwg6urd7qqn0ctx579h9kg0y8qn8gpxw");
+    });
+
+    it("should return transaction from gateway with queryInElastic = false", async () => {
+      const txHash: string = "51ffbf3d27e06fd509c510ef0ff1ea7329359dba89c05aeec333de52a405664d";
+      const results = await transactionGetService.tryGetTransactionFromGateway(txHash, false);
+
+      if (!results) {
+        throw new Error("Properties are not defined");
```
Comment (CatalinFaurPaul) [inline src/test/integration/transaction.get.e2e-spec.ts:81]: misspelled word

File: src/test/integration/transaction.get.e2e-spec.ts (hunk 1 above)
Comment (CatalinFaurPaul) [reply src/test/integration/transaction.get.e2e-spec.ts:64]: fixed

File: src/test/integration/transaction.get.e2e-spec.ts (hunk 2 above)
Comment (CatalinFaurPaul) [reply src/test/integration/transaction.get.e2e-spec.ts:81]: fixed

**Review by tanghel — APPROVED** [review]

**Pull request merged**